### PR TITLE
[Lib-415] Base model tests

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

File: test/lib/baseModelTest.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+  });
+
+  describe('#isNew()', function() {
+
+    it('should be a method of the model', function() {
+      should(model_single()).have.property('isNew').which.is.Function();
+    });
+
+    it('should return correct value', function() {
```
Comment (dancio) [inline test/lib/baseModelTest.js]: u should have tests for `incorrect` value

**Comment by dancio** [comment]
u don't need to mock whole http response but just a `makeRequest` method, lgtm

**Comment by jakubbilko** [comment]
@dancio I thought it would be a more adequate example to just mock the responses in order to chech if the lib wraps them properly into models.

File: test/lib/baseModelTest.js (hunk 1 above)
Comment (jakubbilko) [inline test/lib/baseModelTest.js]: Hm, maybe the description of this example is wrong. Is there a way for the `isNew` method to return an incorrect value? It returns `true` or `false` depending on the existence of the `links` property on the model. Maybe I should leave the description as it is and additionally check the method on a model with the `links` property set?

Comment (dancio) [inline test/lib/baseModelTest.js]: this is exactly what I had in mind we should have tests for both scenarios with `links` which is `false` and without which is `true`

**Comment by dancio** [comment]
@jakubbilko it's fine with me :)

**Pull request merged**